content/articles/sending-sms-twilio-in-flutter/index.md

```diff
@@ -0,0 +1,228 @@
+
+In this article, we are going to develop a new app to illustrate how to send SMS in flutter using Twilio.
+#### Introduction
+
+Flutter is an open source mobile UI framework to build native apps for iOS and Android from a single codebase. It is very powerful if we compare it with react-native or any other cross platform frameworks. One of the main point of flutter app development is, it uses widgets instead views to build the UI. So, the main advantage of flutter is that we can use all widgets like Image,Textbox,Button etc., which are available in Android and iOS platform. Now let's discuss how to send SMS using this Cross Platform Framework.
+### Table of content
+- Introduction
+- App description
+- Key Takeaways 
+- Project Break Down
+	1. The model class to hold the SMS data
+	2. Home Page Widget
+	3. Material App Widget 
+	4. The build method
+- Business Logic and Twilio
+   - Step1 . The Twilio package.
+   - Step2. Required permissions.
+   - Step3 . Sending SMS with update list data.
+   - Step4. Sending SMS from the Database.
+   - Step5. Processing SMS received method.
+   - Step6. Sending messages to user with Twilio:
+ - Conclusion
```

Comment:
Add a "Prerequisites" section here, to provide the audience with any prerequisites that may be required to follow your tutorial.